api/src/pcapi/static/backofficev3/js/addons/pc-postal-address-autocomplete.js

```diff
@@ -473,17 +473,11 @@ class PcPostalAddressAutocomplete extends PcAddOn {
     if ($address) {
       $address.parentElement.classList[isManuelEditing ? 'remove' : 'add']('d-none')
     }
-    if ($latitude && isManuelEditing) {
-      $latitude.value = ''
-      $latitude.disabled = true
-    } else if ($latitude && !isManuelEditing) {
-      $latitude.disabled = false
+    if ($latitude) {
+      $latitude.parentElement.classList[isManuelEditing ? 'remove' : 'add']('d-none')
     }
-    if ($longitude && isManuelEditing) {
-      $longitude.value = ''
-      $longitude.disabled = true
-    } else if ($longitude && !isManuelEditing) {
-      $longitude.disabled = false
+    if ($longitude) {
+      $longitude.parentElement.classList[isManuelEditing ? 'remove' : 'add']('d-none')
```

Comment:
```suggestion
      $longitude.parentElement.classList[isManualEditing ? 'remove' : 'add']('d-none')
```
Je sais que c'est pas le but de la PR, mais si tu pouvais refactor pour mettre `Manual` et pas `Manuel` ça serait super 🙏